ilastik.bat.in

```diff
@@ -25,15 +25,24 @@ rem set more paths
 set QTDIR=%INSTALL_DIR%Qt4
 IF ["%ILASTIK_DIR%"] EQU [""] set ILASTIK_DIR=%INSTALL_DIR%ilastik
 set PYTHONPATH=%ILASTIK_DIR%\lazyflow;%ILASTIK_DIR%\volumina;%ILASTIK_DIR%\ilastik
+set PYTHONHOME=%INSTALL_DIR%\python
 
 rem check if this script was called as 'lastik test'
 if /I "%1" EQU "test" goto :testing
+if /I "%1" EQU "python" goto :python
  
 rem if not, start ilastik normally
 echo Loading ilastik from "%ILASTIK_DIR%"
 "%INSTALL_DIR%python\python" "%ILASTIK_DIR%\ilastik\ilastik.py" %*
 goto :end
 
+:python
+echo ----------------------------
+echo Running python command...
+"%INSTALL_DIR%python\python" %*
```

Comment:
The page you cite says: "SHIFT will not affect the value of %* which holds all the original arguments %1 %2 %3...". So this doesn't work, and I don't know of any better way.